### ISSUE TITLE
Is it possible to share the processed dataset?

### ISSUE THREAD

**Issue by YimingMa** [Vicomtech/DMD-Driver-Monitoring-Dataset#9]
Hi @pncanas333,

Thank you and your team for this remarkable contribution to driver status monitoring.
However, I found it very frustrating to process the raw data you shared via box.com. The overall raw dataset is too large (1.3T), and like issue #6, I have also encountered errors during processing it. To facilitate the research community and also to make this dataset benchmark-able, could you please also provide access to the processed one in your paper (DMD: A Large-Scale Multi-modal Driver Monitoring Dataset for Attention and Alertness Analysis)?

Much appreciated,
Yiming

**Comment by BlockchainRev** [comment]
I agree. The processing script is a bit tough to work with and the dataset is too large to feasibly download, especially from Box.

**Comment by soans1994** [comment]
do we need to pay to create box account? inorder to download this dataset?

**Comment by Yiming-M** [comment]
Hi @pncanas333 and @jdortega,

I understand that the annotations are being continuously updated so it's not very possible to publish the processed dataset. However, can you provide the dBehaviourMD dataset used in your paper *DMD: A Large-Scale Multi-Modal Driver Monitoring Dataset for Attention and Alertness Analysis* instead?

Many thanks,
Yiming

**Comment by pncanas333** [comment]
Hello @Yiming-M,

Apologies for the delayed response, and thank you for your comment. We are currently working on enhancing the dataset and transitioning to a new data storage service to facilitate smoother downloads. We understand the challenges associated with using Box.

Our goal is to ensure the dataset is both comprehensive and user-friendly. More information and detailed instructions will be provided soon.

**Comment by Enigma014** [comment]
Hello @pncanas333 ,

If you have a processed version of the dataset ready by now can you please share a link to it. I assume the size will be smaller and it will be practical to use. I need it for my final year project. Kindly consider my request. Thanks in advance!

**Comment by pncanas333** [comment]
Dear all, 
We have completed the transition from Box to an internal storage service, making it easier to request and receive download links. If you experience delays in the downloading process, it may be due to high demand and multiple download requests. We kindly ask for your patience and suggest trying again.

Additionally, we are offering a mini-sample of the DMD, allowing you to explore the data and become familiar with the dataset without needing to download the entire collection.
